docs/knowledge/ceph/benchmarking.md

```diff
@@ -2,9 +2,182 @@
+                 volume.kubernetes.io/provisioner-deletion-secret-namespace: koor-ceph
+Finalizers:      [kubernetes.io/pv-protection]
+StorageClass:    ceph-block
+Status:          Bound
+Claim:           default/rbd-pvc
+Reclaim Policy:  Delete
+Access Modes:    RWO
+VolumeMode:      Filesystem
+Capacity:        1Gi
+Node Affinity:   <none>
+Message:
+Source:
+    Type:              CSI (a Container Storage Interface (CSI) volume source)
+    Driver:            koor-ceph.rbd.csi.ceph.com
+    FSType:            ext4
+    VolumeHandle:      0001-0009-koor-ceph-0000000000000002-95c5c941-f1c2-4f46-824e-c78df2d0b5a9
+    ReadOnly:          false
+    VolumeAttributes:      clusterID=koor-ceph
+                           imageFeatures=layering
+                           imageFormat=2
+                           imageName=csi-vol-95c5c941-f1c2-4f46-824e-c78df2d0b5a9
+                           journalPool=ceph-blockpool
+                           pool=ceph-blockpool
+                           storage.kubernetes.io/csiProvisionerIdentity=1695737390857-5583-koor-ceph.rbd.csi.ceph.com
+```
+
+
+run common fio tests some of them are mentioned below using the pool and image name from above:
+
+```
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=4 --rw=read --bs=4k --iodepth=32 --fsync=32 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=16 --rw=read --bs=4k --iodepth=32 --fsync=32 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=4 --rw=read --bs=4k --iodepth=16 --fsync=16 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=16 --rw=read --bs=4k --iodepth=16 --fsync=16 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=4 --rw=write --bs=4k --iodepth=32 --fsync=32 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=16 --rw=write --bs=4k --iodepth=32 --fsync=32 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=4 --rw=write --bs=4k --iodepth=16 --fsync=16 --runtime=300 --time_based --group_reporting
+fio --name=test-1 --ioengine=rbd --pool=rbd --rbdname=test-image --numjobs=16 --rw=write --bs=4k --iodepth=16 --fsync=16 --runtime=300 --time_based --group_reporting
+```
+
+unmount the rbd volume when done
+
+```
+umount /tmp/rook-volume
+rbd unmap /dev/rbd0
+```
+
+**rados and rbd bench**
+
+you can use rbdbencher.sh script to do rados and rbd bench tests checkout the tool here: https://github.com/koor-tech/rbdbencher 
+
+**cosbench for object store performance**
+
+For rgw performance testing one can make use of cosbench:
+https://github.com/ceph/cbt/blob/master/docs/cosbench.README 
+
+**TLDR Recommended tools**:
```

Comment:
If it is truly "too long, didn't read," the reader won't make it this far. Might be better without TLDR.